### PR TITLE
[ALLUXIO-2999] Follow one level of symlinks

### PR DESCRIPTION
**Pull request by aaudiber**
https://alluxio.atlassian.net/browse/ALLUXIO-2999

### CODE REVIEW

File: integration/fuse/bin/alluxio-fuse.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
-SCRIPT_DIR="$(cd "$(dirname "$0")"; pwd)"
+SCRIPT_DIR="$(cd "$(dirname "$(realpath "$0")")"; pwd)"
```
Comment (calvinjia) [inline integration/fuse/bin/alluxio-fuse.sh]: `readlink -f`?

Comment (calvinjia) [inline integration/fuse/bin/alluxio-fuse.sh]: oh okay, is `realpath` supported on most OS?

Comment (aaudiber) [inline integration/fuse/bin/alluxio-fuse.sh]: Not OS X

Comment (aaudiber) [inline integration/fuse/bin/alluxio-fuse.sh]: Sorry, meant to use `readlink`, fixed now

**Pull request merged**